cli/src/pcluster/templates/cw_dashboard_builder.py

```diff
@@ -219,6 +229,116 @@ def _add_conditional_storage_widgets(
                 widgets_list.append(graph_widget)
         return widgets_list
 
+    def custom_pcluster_metric_filter(self, metric_name, filter_pattern, custom_namespace):
+        """Adding custom metric filter from named tuple."""
+        metric_filter = logs.CfnMetricFilter(
+            scope=self.stack_scope,
+            id=metric_name + " Filter",
+            filter_pattern=filter_pattern,
+            log_group_name=self.cw_log_group_name,
+            metric_transformations=[
+                logs.CfnMetricFilter.MetricTransformationProperty(
+                    metric_namespace=custom_namespace,
+                    metric_name=metric_name,
+                    metric_value="100",
```

Comment:
This should be
```
metric_value="1",
```

Reply:
yeah but the value is sometimes too small, value still not even 1 in the actual graph